pages/learn/[slug].page.tsx

```diff
@@ -4,9 +4,10 @@ import StyledMarkdown from '~/components/StyledMarkdown';
 import { getLayout } from '~/components/Sidebar';
```

Comment:
Hi Dhairya. Great work adding the TOC on this page. However, we must add it to all the docs pages using the sidebar. This means, reference, spec, and the rest of the pages.

Reply:
@DhairyaMajmudar have you fixed this?